Makefile

```diff
@@ -12,31 +12,32 @@ SHELL := bash
 .SHELLFLAGS := -o pipefail -euc
 
 binary := boots
```

Comment:
looks like this isn't used anymore, might consider removing it

Reply:
yep thanks find/replace doesn't make unnecessary things go away :D